modin/pandas/test/dataframe/test_udf.py

```diff
@@ -305,6 +299,21 @@ def test_query(data, funcs):
         df_equals(modin_result, pandas_result)
 
 
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+def test_query_with_local_vars(data):
+    modin_df = pd.DataFrame(data)
+    pandas_df = pandas.DataFrame(data)
+    x = 2  # noqa F841
```

Comment:
There is no test for a global variable, is there? What about a variable that is a function argument?

Reply:
Test with global variable is added.

Can you give me example when variable is a function argument? I'm not sure that I understand this.

Reply:
Something like this:
```python
def func(arg):
    modin_df.query("@arg")
```